readthedocs/profiles/urls/private.py

```diff
@@ -25,3 +25,13 @@
         name='account_advertising',
     ),
 ]
+
+tokens_urls = [
```

Comment:
Why did we add a new list here instead of just putting it in the `urlpatterns`? Could use a comment if there's a reason.

Reply:
I just followed the pattern that we have in other urls.py file.

Although, I think I need to use a different name for the first list and add it to urlpatterns later. This way allow us to decide which URLs to import and define from corporate. Actually, we don't want to publish this new URL in Corporate since there is not going to be API Tokens yet.